### DISCUSSION TITLE
slicer will not cool part above the default of 35%

### DISCUSSION THREAD

**Discussion by cdw2929** [supermerill/SuperSlicer#2335] [Q&A]
no matter what I change the default speed to it will not send a M106 code to match the highest It will go is S33.
actually the only aspects that work on the part cooling page is the start after (x) layers and full speed after(x) layers, nothing else works as intended
Using version 2.3.57.

I get the correct PWM S speeds if I slice with prusaslicer IE full speed is M106 S255
does anyone know what is wrong?

**Comment by supermerill** [comment]
you project file?
